src/components/minimized_email/MinimizedEmail.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import useCustomer from '../../hooks/useCustomer';
+import PropTypes from 'prop-types';
+
+const MinimizedEmail = ({ step, changeButtonText, onChangeStep, isLoggedIn }) => {
+    const { customer } = useCustomer();
```

Comment:
Maybe we make these components into pure components and move the hook to the parent component